xdsl/interactive/get_all_possible_rewrites.py

```diff
@@ -22,6 +25,31 @@ class IndexedIndividualRewrite(NamedTuple):
     rewrite: IndividualRewrite
 
 
+def convert_indexed_individual_rewrites_to_available_pass(
```

Comment:
now that this is in a file that doesn't depend on the UI, could you please add a test that exercises this function, and doesn't depend on any passes in xDSL?